prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -210,7 +211,7 @@ class SettingsFacade(
         clazz: Class<T>,
         name: String,
         organizationName: String? = null,
-    ): Triple<Boolean, String?, JSON?> {
+    ): Triple<Boolean, String?, JSONB?> {
         val input = try {
```

Comment:
JSONB is a more full-featured datatype for JSON in postgres, and for the munging of the settings objects, it's better.